### ISSUE TITLE
LInux/s390x machines not picking up later git

### ISSUE THREAD

**Issue by sxa555** [AdoptOpenJDK/openjdk-infrastructure#595]
Have had to uninstall the system git in order for the openj9 builds (which require the `-C` parameter to work, as it's not picking up the version in `/usr/local/bin` by default. This is different from the `-2` build machine where it's working despite the older system git being installed

**Comment by AdamBrousseau** [comment]
Path issue? We had a machine last week that didn't have /usr/local on the path.

**Comment by sxa555** [comment]
Yes that is the obvious reason, but it's a matter of whether that's the right thing to do ...

**Comment by sxa555** [comment]
https://github.com/AdoptOpenJDK/openjdk-build/blob/master/build-farm/platform-specific-configurations/linux.sh#L32 <<< Explicitcly sets `/usr/bin` at the start on s390x/J9 ... May try removing that and see what happens. The other option is to explicitly remove the git package of course but I'm reluctant to do somethign that could negatively impact people's machines.

**Comment by sxa555** [comment]
Testing via https://ci.adoptopenjdk.net/view/work%20in%20progress/job/SXA-j9s390xpath/
